src/components/Blogs/Blogs.tsx

```diff
@@ -0,0 +1,128 @@
+// @ts-nocheck
```

Comment:
Is there a reason why we need this for the entire file? If not, we should remove it and use @ts-expect-error with explanation if necessary, or resolve the TS errors.